### ISSUE TITLE
MVP Accessibility Issues [dev_mvp branch]

### ISSUE THREAD

**Issue by AdamLeonSmith** [Path-Check/safeplaces-frontend#28]
- [ ] Delete, Delete All, and Zoom button, and the date range label on the filter - do not have sufficient colour contrast to meet WCAG guidelines.  See https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=AxeChrome
- [ ] The <html> element does not have a lang attribute.  This is an issue for screen-readers.

**Comment by AdamLeonSmith** [comment]
All confirmed fixed EXCEPT the date labels, reopened to track this.
